students/mmancini/lesson02/Lesson2GridPrintPart03.py

```diff
@@ -0,0 +1,31 @@
+                    
+def PrintGrid(iNumBlocks, iRowCols):
```

Comment:
```suggestion
def print_grid(num_blocks, row_cols):
```

python uses snake_case naming style